week5/WordGraph.cpp

```diff
@@ -0,0 +1,78 @@
+//
+// Created by julia on 7/6/17.
+//
+
+#include "WordGraph.h"
+
+
+WordGraph::WordGraph(std::vector<std::string> &GivenDictionary) {
+    std::vector<int> connection (0);
+    for (int i = 0; i < 128; ++i) {
```

Comment:
Yes, the standard part, but what about the extended part? A `char` can have 256 possible values, and to make things more fun, `char` is signed or unsigned depending on the compiler.